packages/base-styles/_mixins.scss

```diff
@@ -1,6 +1,73 @@
 @import "./functions";
 @import "./long-content-fade";
 
+/**
+ * Typography
+ */
+
+@mixin heading-small() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-x-small;
+	line-height: $line-height-x-small;
+}
+
+@mixin heading-medium() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-medium;
+	line-height: $line-height-small;
+}
+
+@mixin heading-large() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
```

Comment:
This bit:

```
	font-family: $font-family-headings;
	font-weight: $font-weight-semi-bold;
```

is repeated across all heading mixins.

Perhaps it can be its own mixin which is reused in each of those mixins?